plugins/stack-overflow/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "@backstage/plugin-stack-overflow",
+  "version": "0.0.0",
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "license": "Apache-2.0",
+  "publishConfig": {
+    "access": "public",
+    "main": "dist/index.esm.js",
+    "types": "dist/index.d.ts"
+  },
+  "scripts": {
+    "build": "backstage-cli plugin:build",
+    "start": "backstage-cli plugin:serve",
+    "lint": "backstage-cli lint",
+    "test": "backstage-cli test",
+    "diff": "backstage-cli plugin:diff",
+    "prepack": "backstage-cli prepack",
+    "postpack": "backstage-cli postpack",
+    "clean": "backstage-cli clean"
+  },
+  "dependencies": {
+    "@backstage/core-components": "^0.8.8",
```

Comment:
i'm awfully sorry for the confusion, but you'll have to go through all of the `@backstage` dependencies manually and setting the versions (plus `^`) to exactly the version of the corresponding package in the repo. For example the `@backstage/config` below would have to be `^0.1.14`. This is part of the mess that the previous release caused - after the next release we should have all of this sorted out hopefully. This should sort out the yarn.lock file changes too